README.md

```diff
@@ -27,24 +27,25 @@ This approach makes the set-up experience more streamlined.
 8. Follow the instructions under 'Configuring the local database'.
 9. Create an IoT Hub (S1) on Azure. You'll need to wait for it to be in an active state, it could take a while.
 10. After your IoT hub has started, you can use Visual Studio integration to connect to it.
-11. In Visual Studio (or on the Azure Portal), you can create a new test Edge Device.
-12. Right click on the device and select “Copy Device Connection String” from the Menu.   
+11. In Visual Studio (or on the Azure Portal), you can create a new IoT Edge Device.
```

Comment:
should be Visual Studio Code (fix all occurrences)

Reply:
fixed